cypress-tests/cypress/e2e/PaymentMethodListUtils/Stripe.js

```diff
@@ -0,0 +1,754 @@
+import { getCustomExchange } from "./Common";
+
+const successfulTestCard = "4242424242424242";
+const successful3DSCard = "4000002500003155";
+
+const successfulNo3DSCardDetails = {
+  card_number: "4242424242424242",
+  card_exp_month: "10",
+  card_exp_year: "25",
+  card_holder_name: "morino",
+  card_cvc: "737",
+};
+
+const successfulThreeDSTestCardDetails = {
+  card_number: "4000002500003155",
+  card_exp_month: "10",
+  card_exp_year: "25",
+  card_holder_name: "morino",
+  card_cvc: "737",
+};
+
+const singleUseMandateData = {
+  customer_acceptance: {
+    acceptance_type: "offline",
+    accepted_at: "1963-05-03T04:07:52.723Z",
+    online: {
+      ip_address: "125.0.0.1",
+      user_agent: "amet irure esse",
+    },
+  },
+  mandate_type: {
+    single_use: {
+      amount: 8000,
+      currency: "USD",
+    },
+  },
+};
+
+const multiUseMandateData = {
+  customer_acceptance: {
+    acceptance_type: "offline",
+    accepted_at: "1963-05-03T04:07:52.723Z",
+    online: {
+      ip_address: "125.0.0.1",
+      user_agent: "amet irure esse",
+    },
+  },
+  mandate_type: {
+    multi_use: {
+      amount: 8000,
+      currency: "USD",
+    },
+  },
+};
+
+export const connectorDetails = {
+  pm_list: {
+    PaymentIntent: {
+      RequestCurrencyUSD: {
+        card: successfulNo3DSCardDetails,
```

Comment:
```suggestion
        payment_method_data: {
          card: successfulNo3DSCardDetails,
        },
```

every where below as well as in other config files.